tools/topology/sof-ehl-rt5660.m4

```diff
@@ -0,0 +1,193 @@
+#
+# Topology for ELKHARTLAKE with rt5660 codec
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Elkhartlake DSP configuration
+include(`platform/intel/ehl.m4')
+include(`platform/intel/dmic.m4')
+
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM0 ----> volume -----> SSP0 BE dailink 0
```

Comment:
don't we have capture support over SSP to get the headset to work?

Reply:
@plbossart We do support capture over SSP. There is no headset on EHL RVP. It only supports headphone + MIC. The jacks are separated.

Reply:
I don't see a capture pipeline connected to SSP?

Reply:
pipeline 1 is for ssp playback, pipeline 2 is for ssp capture. pipeline 3 is for dmic48k.